File: isLucky.java
boolean isLucky(int n) {
    String numero = String.valueOf(n);	    
    char[] digitos = numero.toCharArray();
	int suma1 = 0, suma2 = 0;
	for(int x=0;x<digitos.length/2;x++){
	    suma1 += Integer.parseInt(String.valueOf(digitos[x]));
	    suma2 += Integer.parseInt(String.valueOf(digitos[digitos.length - (x+1)]));
	}
	return (suma1==suma2);
}

File: isLucky.py
def isLucky(n):
    sn = str(n)
    s = len(sn)//2
    f1 = sn[:s]
    f2 = sn[s:]
    n1, n2=0,0
    for x in range(len(f1)):
        n1 += int(f1[x])
        n2 += int(f2[x])
    return n1==n2